cvat-core/src/frames.js

```diff
@@ -341,13 +341,29 @@
         constructor(size, chunkSize, stopFrame, taskID) {
             this._size = size;
             this._buffer = {};
+            this._contextImage = {};
             this._requestedChunks = {};
             this._chunkSize = chunkSize;
             this._stopFrame = stopFrame;
             this._activeFillBufferRequest = false;
             this._taskID = taskID;
         }
 
+        isContextImageAvailable(frame) {
```

Comment:
@azhavoro Andrey, could you please review part of ``cvat-core`` and ``cvat-data`` related with getting 3D meshes from the server?